jsx/InstrumentForm.js

```diff
@@ -65,6 +65,7 @@ function renderRadioLabels(element, key) {
 
 function renderRadio(element, key, onUpdate, isRequired) {
   return (
+      <div>
```

Comment:
This `div` needs a `key` property. Getting this warning now:
```
Warning: Each child in an array or iterator should have a unique "key" prop. Check the render method of `InstrumentForm`. See https://fb.me/react-warning-keys for more information.
```

You can remove the `key` property from RadioGroupElement and add to the top level `div`.